### ISSUE TITLE
Investigate nuget package health issues

### ISSUE THREAD

**Issue by ericsink** [ericsink/SQLitePCL.raw#570]
I opened the package `SQLitePCLRaw.bundle_e_sqlite3mc` in the NuGet package explorer (see screenshot below) and there are 3 items flagged as missing. However, I don't know how important it is to fix these issue.

![grafik](https://github.com/ericsink/SQLitePCL.raw/assets/11139874/5a03a70a-61c0-4618-b30a-698b79a53983)

_Originally posted by @utelle in https://github.com/ericsink/SQLitePCL.raw/issues/566#issuecomment-1807101451_

**Comment by ericsink** [comment]
These issues would be relevant for all packages in SQLitePCLRaw, not just the sqlite3mc ones.

**Comment by utelle** [comment]
> These issues would be relevant for all packages in SQLitePCLRaw, not just the sqlite3mc ones.

You are right. I did not check **all** _SQLitePCLRaw_ packages, but the ones I checked have these issues, too.
